tests/test_input.py

```diff
@@ -10,2690 +10,2624 @@
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            output_metrics,
+            _,
+        ) = parse_config_file(config_path)
+
+        self.assertEqual(do_grid_search, False)
+        self.assertEqual(grid_objectives, [])
+        self.assertEqual(output_metrics, ["neg_mean_squared_error"])
+
+    def test_config_parsing_relative_input_path(self):
+        """Test that config parsing works as expected with relative input directories."""
+        # make a simple config file that has relative paths
+        values_to_fill_dict = {
+            "experiment_name": "config_parsing",
+            "task": "cross_validate",
+            "train_directory": train_dir,
+            "featuresets": "[['f1', 'f2', 'f3']]",
+            "learners": "['LogisticRegression']",
+            "logs": output_dir,
+            "results": output_dir,
+            "objectives": "['f1_score_macro']",
+        }
 
         config_template_path = config_dir / "test_config_parsing.template.cfg"
-        config_path = fill_in_config_options(config_template_path, values_to_fill_dict, sub_prefix)
-        yield check_config_parsing_value_error, config_path
-
-
-def test_config_parsing_bad_featurenames():
-    """Test that config parsing raises error with badly specified featureset names."""
-    # make a simple config file that has bad feature names
-    values_to_fill_dict = {
-        "experiment_name": "config_parsing",
-        "task": "evaluate",
-        "train_directory": train_dir,
-        "test_directory": test_dir,
-        "learners": "['LogisticRegression']",
-        "featuresets": "[['f1', 'f2', 'f3'], ['f4', 'f5', 'f6']]",
-        "logs": output_dir,
-        "results": output_dir,
-    }
-
-    for fname, sub_prefix in zip(
-        ["['set_a']", "['1', 2]", "set_a", "1"],
-        ["wrong_num_names", "wrong_type_names", "wrong_num_and_type1", "wrong_num_and_type2"],
+
+        config_path = fill_in_config_options(
+            config_template_path, values_to_fill_dict, "mislocated_input_file"
+        )
+
+        (
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            train_path,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+            _,
+        ) = parse_config_file(config_path)
```

Comment:
Especially in cases where you only need one value...
```suggestion
        train_path = parse_config_file(config_path)[37]
```